### PR TITLE
Create navigation-bar.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
<p>This navigation bar has four sections.
    <ul class="list-style-circle">
      <li>hamburger-menu</li>
      <li>logo-section</li>
      <li>search-bar</li>
      <li>Nav-links</li>
    </ul>
    <br/>
  </p>

navBar-screenshot

### CODE REVIEW

**Review by Vivek-Py — APPROVED** [review]
Looks good to me! 🚀

File: Components/navigation-bar.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+.header .logo {
+  font-size: 1.2rem;
+  color: var(--primary-color);
+  font-weight: bolder;
+  text-decoration: none;
+}
+
+.hamburger-menu {
+  padding: 0.5rem;
```
Comment (hinanshisuthar) [inline Components/navigation-bar.css]: Could use 8px here

File: Components/navigation-bar.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  cursor: pointer;
+  font-size: 1.4rem;
+}
+
+.hamburger-menu:hover {
+  color: var(--primary-color);
+}
+
+.header .logo i {
+  margin-right: 0.5rem;
```
Comment (hinanshisuthar) [inline Components/navigation-bar.css]: here too.

File: Components/navigation-bar.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  color: var(--dark-primary-color);
+}
+
+.header .search-form {
+  border-radius: 1rem;
+  height: 2.5rem;
+  min-width: 25rem;
+  padding: 0.2rem;
```
Comment (hinanshisuthar) [inline Components/navigation-bar.css]: Same with here.

**Review by hinanshisuthar — COMMENTED** [review]
Everything looks good. Suggested to use px unit for values <=10 px.

File: Components/navigation-bar.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/navigation-bar.css]: Done

**Pull request merged**